### ISSUE TITLE
Field organization??

### ISSUE THREAD

**Issue by suprk03** [Superstars111/recommender#2]
https://github.com/Superstars111/recommender/blob/10ba7b9a607f68234ed5921aa00de37c67fa67ad/models.py#L18-L40
What does this mean about version specific? Will it be possible to specify in that field "not this version but the other versions are fine"/"this anime yes but the remake no" or something like "Both have different strengths and here are the reasons for each"? Or will those things have to be specified in a notes field?
And should the version specific field not be in "Recommendation" not "Media", so that a Media object can be reused in multiple recommendations?

**Comment by Superstars111** [comment]
Yes, version_specific ought to be in Recommendation instead. I put that in the wrong spot as I was drafting it. 

As for what it means, those are good thoughts. Ideally, we should have ways to mark that, but we'll need to find a good way to code it. Maybe instead of a boolean, it can be, "specifically this version," "specifically not one specific version which is also attached," or "it's complicated, see attached note." Or something along those lines.

**Comment by 4RadiantPearlWillShineUponTheIgnorant** [comment]
Also something to be considered is not just the edge cases of adaptations and whether those are recommended, but also whether something is multiple categories. Unsounded is a comic that stands on its own, and also has optional prose side stories. Paranatural _was_ a webcomic, and the artist had wrist problems, so now he does illustrated prose. Kill Six Billion Demons is a webcomic that effectively has short stories beneath the pages sometimes.
